### PR TITLE
Added export and import function of tables.

### PR DESCRIPTION
**Pull request by skasberger**

### CODE REVIEW

**Comment by Bubblbu** [comment]
Can't upgrade the database

```
sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) relation "doi" already exists
```

**Comment by Bubblbu** [comment]
unfortunately still getting this complaint... `psycopg2.ProgrammingError: relation "doi" does not exist`

database is still "clean" without any other tables.

**Comment by skasberger** [comment]
delete all tables again, maybe it does not work with the old upgrade history manifested in the tables.

**Comment by Bubblbu** [comment]
Had to remove alembic as well. `db upgrade` worked now

File: app/__init__.py [hunk 1]
```diff
@@ -267,27 +338,30 @@ def create_ncbi_urls():
...
 
     result_doi = Doi.query.all()
 
-    for row in result_doi:
+    for row in tqdm(result_doi):
         # send request to Unpaywall API
         url_dict = {}
         doi_url_encoded = urllib.parse.quote(row.doi)
-        url = '	https://api.unpaywall.org/v2/{0}?email={1}'.format(doi_url_encoded, NCBI_EMAIL)
+        url = 'https://api.unpaywall.org/v2/{0}?email={1}'.format(doi_url_encoded, ncbi_email)
```
Comment (Bubblbu) [inline app/__init__.py]: ncbi_email -> email

**Pull request merged**